docs/changehistory/NextVersion.md

```diff
@@ -50,7 +50,7 @@ Example:
       "type": "child-1",
       "label": "Child 1",
       "nestedRules": [{
-        "ruleType": "ChildNodes", // this rule now also returns children for `Child 1.2.1`
+        "ruleType": "ChildNodes",
```

Comment:
I think the paragraph above this example needs some adjustment as well.

Also, maybe it's also worth reverting some of the changes I made to the full stack test in #3416?

Reply:
Removed unnecessary conditions from full stack test ruleset. 
Made an attempt to adjust NextVersion documentation to match current implementation.